rpmfusion-free-rawhide.repo

```diff
@@ -3,6 +3,7 @@ name=RPM Fusion for Fedora Rawhide - Free
 #baseurl=http://download1.rpmfusion.org/free/fedora/development/$basearch/os/
 metalink=https://mirrors.rpmfusion.org/metalink?repo=free-fedora-rawhide&arch=$basearch
 enabled=1
+skip_if_unavailable=1
```

Comment:
This is needed because you have enabled repo_gpgcheck which does NOT work.
Also the fedora (hence ours) way to have the repo content verified is to use metalinks with mirror manager on https.

We are not going to deviate from fedora here.